support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/ticket/refreshtoken/OAuth20DefaultRefreshToken.java

```diff
@@ -21,11 +26,19 @@
  */
 @Entity
 @DiscriminatorValue(OAuth20RefreshToken.PREFIX)
+@Getter
 @NoArgsConstructor
 public class OAuth20DefaultRefreshToken extends OAuth20DefaultCode implements OAuth20RefreshToken {
 
     private static final long serialVersionUID = -3544459978950667758L;
 
+    /**
+     * The ticket ids which are tied to this ticket.
+     */
+    @Lob
+    @Column(name = "DESCENDANT_TICKETS", length = Integer.MAX_VALUE)
+    private HashSet<String> descendantTickets = new HashSet<>(0);
```

Comment:
As noted earlier, these are exclusively access tokens. So please rename the column and the field name to indicate this explicitly.

Reply:
Done